docs/docsite/rst/network/user_guide/network_debug_troubleshooting.rst

```diff
@@ -453,12 +452,25 @@ Suggestions to resolve:
         provider: "{{ cli }}"
         timeout: 30
 
+For network_cli, netconf connection type (applicable from 2.7 onwards):
+
+.. FIXME: Detail error here
+
+Suggestions to resolve:
+
+.. code-block:: yaml
+
+    - name: save running-config
+      ios_command:
+        commands: copy running-config startup-config
+      vars:
+        ansible_command_timeout: 30
+
 Some operations take longer than the default 10 seconds to complete.  One good
 example is saving the current running config on IOS devices to startup config.
 In this case, changing the timeout value form the default 10 seconds to 30
```

Comment:
Sorry didn't notice this earlier - please change 'form' to 'from'.  Otherwise looks good to me.

Reply:
Fixed